### PR TITLE
feat: added new page guidelines

### PR DESCRIPTION
**Pull request by isaackps**
## Summary

Added a new guidelines page

## Changes

- added guidelines pages
- updated accordion item component

## Issues

https://www.pivotaltracker.com/n/projects/2424361/stories/180258594

### CODE REVIEW

File: src/components/Guidelines/GuidelinesContent.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React, { FunctionComponent } from "react";
+import ReactMarkdown from "react-markdown";
+import { AccordionItem } from "../../components/UI/Accordion/AccordionItem";
+import { importAll } from "../../common/utils/importAll";
+import { getSortedByDateAsc } from "../../utils";
+import remarkGfm from "remark-gfm";
```
Comment (cavacado) [inline src/components/Guidelines/GuidelinesContent.tsx]: what do we need from remarkGFM that is not supported from ReactMarkdown?

Comment (isaackps) [reply src/components/Guidelines/GuidelinesContent.tsx]: my bad, should not have this package inside, will double check again

**Comment by cavacado** [comment]
deploy-preview-486--tradetrust-ropsten netlify app_guidelines
dont think you intended the illustration to be that huge.
Screenshot 2021-11-16 at 4 06 14 PM
the navbar okay?

**Comment by isaackps** [comment]
oh wah ok will edit the image, and check with jerry about the navbar

**Comment by isaackps** [comment]
ok fixed the over stretching of the image and nav bar

**Review by cavacado — APPROVED** [review]

**Pull request merged**